### ISSUE TITLE
[Warning] Text elements should have a reasonable contrast ratio (50)

### ISSUE THREAD

**Issue by rodowi** [mxabierto/participa#230]
[Warning] Text elements should have a reasonable contrast ratio (50)
https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_color_01
```html
<a href="http://www.gob.mx/consulta" target="_self">
  <i class="icon icon-home"></i> Inicio
</a>
```
*** y otros 50 más

**Comment by rodowi** [comment]
Encontrado en:
- /consulta
-  /consulta/auth/login

**Comment by codemoe** [comment]
@rodowi Éste tema tendría un impacto directo sobre el diseño de la plataforma. Vale la pena ver exactamente cuales son los 50 elementos que muestran este programa para luego evaluar que ajustes podemos hacer al color de estos elementos

**Comment by rodowi** [comment]
ok, esto lo comentaré con el equipo de gob.mx, mientras lo mando al backlog

**Comment by rodowi** [comment]
#393C3E
